kcidb/oo/__init__.py

```diff
@@ -6,9 +6,11 @@
 from abc import ABC, abstractmethod
 from functools import reduce
 from cached_property import cached_property
-import kcidb.db
-from kcidb.misc import LIGHT_ASSERTS
 from kcidb.orm import Type, SCHEMA, Pattern, Source
+from kcidb.misc import LIGHT_ASSERTS
+import kcidb.misc
+import kcidb.orm
+from . import argparse
```

Comment:
We don't do relative imports in general, let's not introduce them piecemeal, and keep using the absolute imports for now. If we're going to change that, then it's better to come up with a general scheme and change everything at once. Uniformity makes code easier to read.